command/tls_cert_create.go

```diff
@@ -294,3 +300,37 @@ func (c *TLSCertCreateCommand) Run(args []string) int {
 
 	return 0
 }
+
+func recordPreparation(certType string, regionName string, domain string, dnsNames []string, ipAddresses []net.IP) ([]net.IP, []string, string, []x509.ExtKeyUsage, string) {
+	var (
+		extKeyUsage             []x509.ExtKeyUsage
+		name, regionUrl, prefix string
+	)
+	if certType == "server" || certType == "client" {
+		extKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth}
+		ipAddresses = append(ipAddresses, net.ParseIP("127.0.0.1"))
+	} else if certType == "cli" {
+		extKeyUsage = []x509.ExtKeyUsage{}
```

Comment:
Since the CLI's HTTP client uses this certificate for mTLS, I think we should have the `x509.ExtKeyUsageClientAuth` here too, but I don't think it matters functionally.